pylabrobot/pumps/backend.py

```diff
@@ -19,8 +20,47 @@ def run_continuously(self, speed: float):
     If speed is 0, the pump will be halted.
 
     Args:
-      speed: speed in rpm.
+      speed: speed in rpm/pump-specific units.
```

Comment:
I'm not a huge fan of having a parameter on the front end / abstract backend be something that's machine specific. What's the unit for the agrow pumps again?

Reply:
mL/second. In all liquid pumping I've worked with that has been the standard unit

Reply:
for most pump implementations that don't have some kind of built-in tachometer/tracking of the number of rotations it will be volume per unit time, which is our implementation assumes unit volume per second

Reply:
Ok. I will add a `tube_volume` attribute to Masterflex, so we can do the volume/time -> rotations/time (rpm) conversion on that backend.

Reply:
Okay, that sounds good